roles/bind-keycloak-apb/templates/client_configmap.j2

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ keycloak_configmap_name }}-{{ consumer }}
+  namespace: {{ namespace }}
+  labels:
+    mobile: enabled
+    serviceName: keycloak
+    consumerID: {{ consumer }}
```

Comment:
name it `consumerId: {{ consumerId }}` to be consistent. If thats consumer name then it stay `consumer`